spec/open_api_spec.rb

```diff
@@ -1,4 +1,5 @@
 describe "OpenAPI stuff" do
+  SKIP_TABLES = %w[tenants schema_migrations ar_internal_metadata rbac_seeds portfolio_tags portfolio_item_tags images icons service_plans].freeze
```

Comment:
@syncrou Can you describe the criteria for selecting with tables we skip?  Some of these make sense as they are internal but not sure all apply.

Reply:
They are tables not exposed on the openapi.json schema. The reasons for that are many.  Mostly internal models that will never be accessed from the outside.